_pages/policies/responding-to-public-disclosure-vulnerabilities.md

```diff
@@ -49,6 +49,8 @@ The first responder will communicate directly with the Infrastructure Lead, who
 
 ## Respond to reporter
 
+HackerOne will provide responses to reports received via the H1 platform. For reports received via email or webform, refer to the following guidance:
```

Comment:
One point: this is true _for issues reported against the bug bounty scope_ (e.g. Federalist, cloud.gov, etc), but not for issues that fall under the general VDP. Our procurement doesn't cover triage services for those, so for reports that aren't in that scope, we'll need to do both triage and validation.